packaging/conda/meta.yaml

```diff
@@ -81,7 +83,7 @@ outputs:
 
   # kni-transfer package (designed only to test kni)
   - name: kni-transfer
-    version: {{ os.environ.get('KHIOPS_VERSION') }}
+    version: {{ version.replace("-", ".") }}
```

Comment:
I would substitute "-" for "" to be consistent with current versioning scheme (see comment above).